### ISSUE TITLE
Disabling the custom layout dialog

### ISSUE THREAD

**Issue by durga2112** [Astroua/CARTAvis#97]
When setting a custom layout from the scripted client, the custom layout dialog stays on the screen. Here is some Python code that can demonstrate the issue:

```python
import cartavis

# Adjust the parameters here to suit your own system
v = cartavis.Cartavis('/home/jeff/scratch/build/dev/cpp/desktop/desktop', '/home/jeff/.cartavis/config.json', 9999, '/home/jeff/dev/CARTAvis/carta/VFS/DesktopDevel/desktop/desktopIndex.html', '/home/jeff/CARTA/Images/mexinputtest.fits')

v.setCustomLayout(2,1)
v.setPlugins(['CasaImageLoader', 'Histogram'])
```

At this point, the dialog for the custom layout is still on the screen, but the OK and Cancel buttons don't do anything and I can't see a way to dismiss the dialog at all. The real issue is that this dialog need not be shown at all when setting a layout from the scripted client. I notice that in Application.js there is some code to show the popup only if a session is not being restored. Can that condition be made a little more general so that the popup also wouldn't be shown when setting a layout from the scripted client?

**Comment by durga2112** [comment]
I have noticed another oddity with the session restore and custom layout dialogs. Try this in the GUI:

- bring up the Session -> Manage/Restore... dialog
- close it
- try to bring up the Layout -> Custom Layout dialog

On both of my machines, the Custom Layout dialog does not appear. If this is actually a separate issue, let me know; I have only included it in a comment here because it seems to me like it's all related.
